open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-
-## Building our Docassemble interview
 
-### What kinds of information do we need to gather?
-
-Before we start, let's think about what kinds of information we will want to ask.
-
-This Docassemble interview will use two kinds of screens:
-
-1. Yes/no questions
-2. Ending screens
-
-As a reminder, ending screens use the `event` specifier. 
-
-We haven't seen a `question` block that uses just a yes/no question yet. Here's a [link to the documentation](https://docassemble.org/docs/questions.html#tocAnchor-1-6-1).
-
-Let's take a look at an example:
+We'll need two ending screens. You make endings screens with an [`event` attribute](https://docassemble.org/docs/questions.html#event) in a `question` block. That's like the `field` attribute we've used before. Unlike regular question screens, your user isn't going to give an answer here, so there's no variable for Docassemble to find. The `event` attribute tells Docassemble how to find the question without needing a variable name.
 
 ```yaml
+---
+id: no_go_advice
+event: dont_go_to_party
+question: |
+  Don't go to that dumb party. Do something fun instead!
+---
+id: go_advice
+event: go_to_party
 question: |
-  Are you at least 18 years of age?
-yesno: over_eighteen
+  You should go to the party! I promise it'll be awesome!
+---
 ```
 
-And here's what it looks like when we run it:
-
-<img alt="Yes/No question" src={useBaseUrl('img/Yes_no.png')} />
+### What do you need to know?
 
-Docassemble uses two different kinds of `question` blocks. This new kind only lets you ask for one variable on a screen, and doesn't use the `fields` specifier we learned earlier. It's kind of a simplified shorcut version.
+Well, without user research, you don't really know what good questions would be. You don't have funding or time to do that discovery work, though, but you don't care that much about those strangers and it's not like they have other options, so you'll start by making a guess and worry about it later.
 
-Let's look at line 3 of our sample:
+Do they even like parties? That seems like a place to start.
 
 ```yaml
-yesno: over_eighteen
+---
+id: does_like_parties
+question: |
+  Do you like parties?
+fields:
+  - Do you like parties: user_likes_parties
```

Comment:
So this is a point off for me. My `question` and my actual question are the same text. My bad for not trying out these questions in an actual script.

Thinking about it more deeply, the way Docassemble sets up these structures is a bit suboptimal for learning. There are two kinds of particles here and each of them has their own weirdness that has to be learned. If you use `yesno`, you have to learn another kind of particle because it's different than `fields`, but if you use `fields` you either have to make an empty title or add `nolabel` (or whatever it is) and learn _that_ particle/quirk. I think I might go for an empty title here? Dang.

Any thoughts on other options?

Reply:
I think using `no label` will be understandable here. You're right that waiting to introduce `yesno` question blocks makes sense, and `no label` is a smaller thing to learn.